apps/frontend/package.json

```
@@ -11,7 +11,7 @@
 		"lint": "eslint -c ./.eslintrc.cjs --ignore-path ./.eslintignore \"./**/*.{ts,tsx,cjs}\"",
 		"type-check": "tsc --project ./tsconfig.json",
 		"depcheck": "depcheck --config ./.depcheckrc.json",
-		"stylelint": "stylelint --config ./stylelint.config.cjs \"./src/**/*.scss\"",
+		"stylelint": "stylelint --config ./stylelint.config.cjs \"./src/**/*.scss\" --fix",
```

as practice, add another comment with fix and call it - `stylelint:fix`
```suggestion
		"stylelint": "stylelint --config ./stylelint.config.cjs \"./src/**/*.scss\"",
```